internal/v4/api_test.go

```diff
@@ -898,6 +898,20 @@ func (s *APISuite) TestExtraInfoPutUnauthorized(c *gc.C) {
 		ExpectStatus: http.StatusProxyAuthRequired,
 		ExpectBody:   dischargeRequiredBody,
 	})
+	httptesting.AssertJSONCall(c, httptesting.JSONCallParams{
```

Comment:
This should probably be asserting that the response header contains the appropriate WWW-Authorize header. Mind you, it's not that crucial as the tests failed anyway when it didn't.